### ISSUE TITLE
Dosen't seem to be working

### ISSUE THREAD

**Issue by HashRaygoza** [oslego/better-twitter#3]
Even after cleaning cache and cookies all the twitter anoyances still show, this is with v1.4 of the extension and firefox 69.0.3 on Windows 10

![Captura de pantalla 2019-10-18 11 11 11](https://user-images.githubusercontent.com/19654490/67110560-abbb1b80-f198-11e9-92fb-5079ee1a896b.png)
![Captura de pantalla 2019-10-18 11 11 23](https://user-images.githubusercontent.com/19654490/67110567-aeb60c00-f198-11e9-8b2c-32272a1932b6.png)

**Comment by oslego** [comment]
Indeed, you're right. The issue comes from the language of the Twitter UI which looks like Spanish in the screenshots. Some of the features do work, like hiding follower count, likes count, etc. via the "No Fame" and "No Vanity" options. Their functionality is independent of language.

However, hiding the "Who To Follow", "Trends for You", and footer boxes do not work when the Twitter UI is in any other language than English.

This sucks. But the solution isn't simple. 

**Context:**
After the recent Twitter interface redesign, the underlying markup of the page was turned into a jumbled obfuscated mess. There's no guarantee the CSS class names will be stable and IDs are missing. 

Unfortunately, the most reliable way to identify some of the elements like "Trends for You" and "Who to Follow" boxes in this interface is via `aria-label` attributes. These are used as an accessibility helper for screen reader software and their values are language-dependent (they will be read out aloud to users of screen readers). The extension is now [configured for English](https://github.com/oslego/better-twitter/blob/master/content.css#L21,L25). Wherever it makes use of `aria-label` values, it will fail to match the elements in other languages. 

The solution following this pattern isn't simple. It would involve creating selectors matching the correct language term for all supported languages. That's beyond my capabilities. I already consider this a fragile matching solution. Strict element tree path matches is an equally fragile method because the DOM structure can and does change frequently. If you have other ideas, I'm open to discuss and accept a pull request with a fix.

I will keep this issue open, since it is an issue, but I will retitle it to match. 

Until there's a more robust way to target those boxes without using `aria-label` values, I'm afraid the only working options across all Twitter UI languages will be "No Fame" and "No Popularity".

**Comment by oslego** [comment]
While this is not ideal, you can get all the options working by switching the Twitter UI interface to English. 

The Twitter help docs explain how to change the language: https://help.twitter.com/en/managing-your-account/how-to-change-language-settings

**Comment by HashRaygoza** [comment]
I can do that, Thank you :)
